### ISSUE TITLE
Stuck in retry loop after failed payment (Android)

### ISSUE THREAD

**Issue by Puruchandra** [razorpay/razorpay-flutter#170]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
Please provide a clear and concise description of what the bug is. Include screenshots if needed.
Please test using the latest Razorpay Flutter plugin release to make sure your issue has not already been fixed: 

## Cocoapod Version : 1.10.1
   razorpay-pod (1.1.12)
   razorpay_flutter (1.1.10):

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. open checkout
2. fail the payment

## Expected Results
After failing the payment, it retries again and basically stuck in loop of retries, have to manually cancel the payment from top right. This issue is only happening in Android. ios works fine. 

## Snack, code example, screenshot, or link to a repository:

void razrpayOpenCheckout(od.OrderModel order) async {
    var options = {
      'key': order.transactionDetails["rpay_key_id"],
      'amount': order.transactionDetails["amount"],
      'retry': {'enabled': false},
      'order_id': order.transactionDetails["id"],
      'external': {
        'wallets': ['paytm']
      }
    };

    try {
      _razorpay.open(options);
    } catch (e) {
      Navigator.pop(context);
    }
  }

**Comment by sumedht** [comment]
@Puruchandra u have to pass above values for retry
[https://razorpay.com/docs/payment-gateway/android-integration/standard/checkout-form-fields/](https://razorpay.com/docs/payment-gateway/android-integration/standard/checkout-form-fields/)

**Comment by Puruchandra** [comment]
> @Puruchandra u have to pass above values for retry
> https://razorpay.com/docs/payment-gateway/android-integration/standard/checkout-form-fields/

@sumedht yes but i dont want it to retry, its retrying automatically on its own even after it is set to false

**Comment by nadeem-golflan** [comment]
@sumedht i am also facing the same issue. When the transaction fails I have to manually close the payment gateway.

**Comment by Puruchandra** [comment]
@sumedht please look into it
